<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleManagerWindow</class>
 <widget class="QWidget" name="ModuleManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="engine.qrc">
    <normaloff>:/engine/images/modules.png</normaloff>:/engine/images/modules.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="Filter">
     <property name="leftMargin">
      <number>26</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QLineEdit" name="FilterText">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearFilter">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="engine.qrc">
         <normaloff>:/engine/images/close_gray.png</normaloff>:/engine/images/close_gray.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>26</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>13</number>
     </property>
     <item>
      <widget class="QScrollArea" name="ScrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>658</width>
          <height>565</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="MainWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>20</number>
              </property>
              <property name="verticalSpacing">
               <number>10</number>
              </property>
              <item row="3" column="0">
               <widget class="QLabel" name="LabelNativeModules">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Native modules:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="margin">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="LabelEmptyCustom">
                <property name="styleSheet">
                 <string notr="true">*{margin-left:20px}</string>
                </property>
                <property name="text">
                 <string>Empty. Use &quot;Install module&quot; button to add.</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="NotFoundCustom">
                <property name="styleSheet">
                 <string notr="true">*{margin-left:20px}</string>
                </property>
                <property name="text">
                 <string>Not found</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="LabelCustomModules">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Custom modules:</string>
                </property>
                <property name="margin">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="NotFoundNative">
                <property name="styleSheet">
                 <string notr="true">*{margin-left:20px}</string>
                </property>
                <property name="text">
                 <string>Not found</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{background-color:gray}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{background-color:#444}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>26</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>13</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QPushButton" name="CreateNewModule">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string> Create new module</string>
        </property>
        <property name="icon">
         <iconset resource="engine.qrc">
          <normaloff>:/engine/images/crud_create.png</normaloff>:/engine/images/crud_create.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InstallModule">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string> Install module</string>
        </property>
        <property name="icon">
         <iconset resource="engine.qrc">
          <normaloff>:/engine/images/install.png</normaloff>:/engine/images/install.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>203</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Close">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>28</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string> Close</string>
        </property>
        <property name="icon">
         <iconset resource="engine.qrc">
          <normaloff>:/engine/images/stop.png</normaloff>:/engine/images/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="engine.qrc"/>
 </resources>
 <connections/>
</ui>
